src/components/IdentityView/IdentityView.tsx

```diff
@@ -221,7 +233,7 @@ class IdentityView extends React.Component<Props, State> {
 
   private toggleContacts() {
     const { contacts, myIdentity } = this.props
-
+    let title = 'Add Contact Favourites'
```

Comment:
Question:
Where is this used?

Reply:
Woops, I was trialling something. Thank you